modules/rest-util/src/blaze/handler/util.clj

```diff
@@ -55,53 +55,59 @@
     "exception"))
 
 
-(defn operation-outcome
-  [{:fhir/keys [issues issue operation-outcome]
+(defn- operation-outcome-issues [issues category]
+  (map (fn [{:fhir.issues/keys [severity code diagnostics expression]}]
+         (cond->
+           {:severity "error"
+            :code (or code (issue-code category))}
+           severity
+           (assoc :severity severity)
+           diagnostics
+           (assoc :diagnostics diagnostics)
+           (coll? expression)
+           (assoc :expression expression)
+           (and (not (coll? expression))
+                (some? expression))
+           (assoc :expression [expression])
+           )) issues))
+
+
+(defn- operation-outcome-issue
+  [{:fhir/keys [issue operation-outcome]
     :fhir.issue/keys [expression]
     :blaze/keys [stacktrace]
     ::anom/keys [category message]}]
+  [(cond->
```

Comment:
I would move the vector out, because the method is in singular ans do should only return one issue